spec/Modules.tex

```diff
@@ -220,40 +226,57 @@ \section{Access of Module Contents}
 depending on the visibility of the module
 itself~(\rsec{Visibility_Of_A_Module}) and the visibility of each
 individual symbol~(\rsec{Visibility_Of_Symbols}).  This can be done
-via explicit naming~(\rsec{Explicit_Naming}) or the use
-statement~(\rsec{Using_Modules}).
+via the use statement~(\rsec{Using_Modules}) or qualified
+naming~(\rsec{Explicit_Naming}).
 
 \subsection{Visibility Of A Module}
 \label{Visibility_Of_A_Module}
 \index{modules!access}
 
-A module defined at file scope is visible anywhere. The visibility of a nested
-module is subject to the rules of~\rsec{Visibility_Of_Symbols}. There,
-the nested module is considered a "symbol defined at the top level
-scope" of its outer module.
+A top-level is available for use~(\rsec{Using_Modules}) anywhere. The
```

Comment:
You omitted "module" here, I think it should read "A top-level module is available..."

Reply:
Sheesh, thanks!